### DISCUSSION TITLE
Focus after closing a modal

### DISCUSSION THREAD

**Discussion by jeana-adhoc** [uswds/uswds#5890] [Accessibility] [answered]
I believe the correct behavior after you close a modal (and you remain on the page that triggered the modal) is that focus should return to the item that triggered the modal.  Is this correct? The [modal here in USWDS](https://designsystem.digital.gov/components/modal/) seems to do that.  If this is correct, can you add a note about this (expected?) behavior into the guidance (And any hints as to how to do this if possible). 

If that behavior isn't expected -- is there a general guidance on what to do after a modal is closed?

Thanks!
- Jeana

**Comment by amycole501** [comment, marked as answer]
You're correct! We are working on the accessibility audit and checklist page which is going live in the coming months. This can be added to the Focus Order check 2.4.3 as an expected behavior.
